### PR TITLE
Updated experiment files

### PR DESCRIPTION
**Pull request by subutai**

### CODE REVIEW

File: projects/imagenet/experiments/base.py [hunk 1]
```diff
@@ -38,7 +38,7 @@
 # See http://github.com/pytorch/examples/blob/master/imagenet/main.py
 DEFAULT = dict(
     # Results path
-    local_dir=os.path.expanduser("~/nta/results/imagenet"),
+    local_dir=os.path.expanduser("~/nta/results/experiments/imagenet"),
```
Comment (lscheinkman) [inline projects/imagenet/experiments/base.py]: Why the extra "experiments" folder?

File: projects/imagenet/experiments/custom_super.py [hunk 2]
```diff
@@ -176,14 +179,21 @@ def my_auto_sparse_linear_params(input_size, output_size):
...
 SUPER_SPARSE100_LONG.update(dict(
-    epochs=80,
+    epochs=60,
+    checkpoint_freq=1,
+    keep_checkpoints_num=2,
+    checkpoint_score_attr="training_iteration",
+    checkpoint_at_end=True,
+
+    # Seed
+    seed=tune.sample_from(lambda spec: np.random.randint(2, 10000)),
```
Comment (lscheinkman) [inline projects/imagenet/experiments/custom_super.py]: `tune.sample_from` without  `num_samples` is just one random seed. Was that your intention here?

File: projects/imagenet/experiments/sparse_r1.py [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    )),
+
+    lr_scheduler_args=dict(
+        max_lr=6.0,
+        div_factor=6,  # initial_lr = 1.0
+        final_div_factor=4000,  # min_lr = 0.00025
+        pct_start=5.0 / 120.0,
+        epochs=120,
+        anneal_strategy="linear",
+        max_momentum=0.01,
```
Comment (lscheinkman) [inline projects/imagenet/experiments/sparse_r1.py]: `max_momentum` is not used when `cycle_momentum` is false

File: projects/imagenet/experiments/sparse_r1.py [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+    # Use a higher learning rate and no momentum for sparse superconvergence
+    lr_scheduler_class=torch.optim.lr_scheduler.OneCycleLR,
+    lr_scheduler_args=dict(
+        max_lr=6.0,
+        div_factor=6,  # initial_lr = 1.0
+        final_div_factor=4000,  # min_lr = 0.00025
+        pct_start=5.0 / 60.0,
+        epochs=60,
+        anneal_strategy="linear",
+        max_momentum=0.01,
```
Comment (lscheinkman) [inline projects/imagenet/experiments/sparse_r1.py]: `max_momentum` is not used when `cycle_momentum` is false

File: projects/imagenet/experiments/base.py (hunk 1 above)
Comment (subutai) [reply projects/imagenet/experiments/base.py]: I am adding another layer, to keep some other stuff also under the results directory.

File: projects/imagenet/experiments/sparse_r1.py (hunk 4 above)
Comment (subutai) [reply projects/imagenet/experiments/sparse_r1.py]: ok. I feel better leaving it in more explicitly though 😄
